sw/airborne/firmwares/rotorcraft/navigation.h

```diff
@@ -59,6 +59,8 @@ extern float nav_radius;
 extern int32_t nav_leg_progress;
 extern uint32_t nav_leg_length;
 
+extern bool_t nav_survey_active;
```

Comment:
since you made it a module, why do you keep a dependency to the general navigation ?